src/Akka.Persistence.Hosting/JournalOptions.cs

```diff
@@ -0,0 +1,85 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="JournalOptions.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Text;
+using Akka.Configuration;
+using Akka.Hosting;
+
+#nullable enable
+namespace Akka.Persistence.Hosting
+{
+    public abstract class JournalOptions
+    {
+        private readonly bool _isDefault;
+        
+        protected JournalOptions(bool isDefault)
+        {
+            _isDefault = isDefault;
+        }
+        
+        public abstract string Identifier { get; set; }
+        
+        /// <summary>
+        ///     <para>
+        ///         Should corresponding journal table be initialized automatically.
+        ///     </para>
+        ///     <b>Default</b>: false
+        /// </summary>
+        public bool AutoInitialize { get; set; } = false;
```

Comment:
Does this also apply to Azure / Redis / MongoDb etc? Or is this setting ignored on platforms that don't support it?

(I'd just make that clear in the XML-DOC)